src/ring_theory/unique_factorization_domain.lean

```diff
@@ -273,7 +273,7 @@ iff.intro
     rintros ⟨⟨c⟩, eq⟩,
     have : c ≠ 0, from (mt mk_eq_zero.2 $
       assume (hc : quot.mk setoid.r c = 0),
-      have (0 : associates α) ∈ q, from prod_eq_zero_iff.1 $ eq ▸ hc.symm ▸ mul_zero _,
+      have (0 : associates α) ∈ q, from prod_eq_zero_iff.1 $ eq.symm ▸ hc.symm ▸ mul_zero _,
```

Comment:
I got confused what `eq.symm ▸ _` means here: could you rename the local variable `eq` so that it's clearer that this is dot notation of that variable, not `_root_.eq.symm`?